cmd/tsconnect/wasm/wasm_js.go

```diff
@@ -31,11 +31,12 @@ import (
 	"tailscale.com/ipn/ipnlocal"
 	"tailscale.com/ipn/ipnserver"
 	"tailscale.com/ipn/store/mem"
+	"tailscale.com/logpolicy"
```

Comment:
```suggestion
	"tailscale.com/logpolicy"hidden
```

Reply:
This pull request was submitted 8 days ago. There isn't a reason to keep commenting on merge pull requests.

Also, this suggestion doesn't even compile.